src/sentry/static/sentry/app/views/settings/organizationFiltersAndSampling/modals/transactionRuleModal.tsx

```diff
@@ -0,0 +1,164 @@
+import CheckboxFancy from 'app/components/checkboxFancy/checkboxFancy';
+import ExternalLink from 'app/components/links/externalLink';
+import {t, tct} from 'app/locale';
+import space from 'app/styles/space';
+import {Organization} from 'app/types';
+import {
+  DynamicSamplingConditionOperator,
+  DynamicSamplingRule,
+} from 'app/types/dynamicSampling';
+import {defined} from 'app/utils';
+import Field from 'app/views/settings/components/forms/field';
+import Form from 'app/views/settings/components/forms/form';
+import NumberField from 'app/views/settings/components/forms/numberField';
+import SelectField from 'app/views/settings/components/forms/selectField';
+
+import MatchField from './matchField';
+
+const conditionChoices = [
+  [DynamicSamplingConditionOperator.ALL, t('All Transactions')],
+  [DynamicSamplingConditionOperator.GLOB_MATCH, t('Releases')],
+  [DynamicSamplingConditionOperator.STR_EQUAL_NO_CASE, t('Enviroments')],
+  [DynamicSamplingConditionOperator.EQUAL, t('Users')],
+];
+
+type Props = ModalRenderProps & {
+  organization: Organization;
+  onSubmit: (rule: DynamicSamplingRule) => void;
+};
+
+type State = {
+  tracing: boolean;
+  condition: DynamicSamplingConditionOperator;
+  match: string;
+  sampleRate?: number;
+};
+
+class TransactionRuleModal extends React.Component<Props, State> {
+  state: State = {
+    tracing: true,
+    condition: DynamicSamplingConditionOperator.ALL,
+    match: '',
+  };
+
+  handleSubmit = async () => {
+    const {sampleRate} = this.state;
+
+    if (!defined(sampleRate)) {
+      return;
+    }
+
+    // TODO(PRISCILA): Finalize this logic according to the new structure
+  };
+
+  handleSubmitSuccess = () => {};
+
+  handleClickTracing = () => {
+    this.setState(prevState => ({tracing: !prevState.tracing}));
+  };
+
+  handleChange = <T extends keyof State>(field: keyof State, value: State[T]) => {
+    if (field === 'sampleRate') {
+      this.setState(prevState => ({
+        ...prevState,
+        sampleRate: value ? Number(value) : undefined,
+      }));
+      return;
+    }
+    this.setState(prevState => ({...prevState, [field]: value}));
+  };
+
+  render() {
+    const {Header, Body, closeModal} = this.props;
+    const {tracing, condition, sampleRate} = this.state;
+
+    const submitDisabled = !defined(sampleRate);
+
+    return (
+      <React.Fragment>
+        <Header closeButton onHide={closeModal}>
+          {t('Add a custom rule for transactions')}
+        </Header>
+        <Body>
+          <Form
+            submitLabel={t('Save')}
+            onCancel={closeModal}
+            apiEndpoint=""
+            onSubmit={this.handleSubmit}
+            initialData={{condition}}
+            onFieldChange={this.handleChange as Form['props']['onFieldChange']}
+            submitDisabled={submitDisabled}
+            requireChanges
+          >
+            <Field
+              label={t('Tracing')}
+              help={t('this is a description')}
```

Comment:
Is this copy, and the other description fields coming soon?

Reply:
it will come soon 😉